app/controllers/api/v1/searches_controller.rb

```diff
@@ -34,6 +34,10 @@ def scrape_from_sites(website)
       facade = AllRecipesScrapeFacade.new
       recipe_results = facade.recipe_details(website)
       render json: WebScrapeSerializer.recipes(recipe_results), status: 200
+    elsif search_term.include?('https://www.foodnetwork.com')
```

Comment:
do you think it's better to leave https://www. out to be consistent with the other elsif statement?

Reply:
yes let me change that